### PR TITLE
Resize cheat sheet to fit content

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
Issue #15

### CODE REVIEW

File: src/cheat_sheet.py [hunk 1]
```diff
@@ -56,17 +47,36 @@ def close(self):
         self.canvas.unregister("mouse", self.mouse)
         self.canvas.close()
 
+    def debounce_resize(self, width: int, height: int):
```
Comment (pokey) [inline src/cheat_sheet.py:50]: Presumably this is because draw gets called multiple times in quick succession?  Might be worth a comment to say why

**Review by pokey — APPROVED** [review]
This looks great! Left one minor comment

**Comment by pokey** [comment]
So for the issue with cheat sheet potentially being bigger than user's screen size entirely, I guess we could have a scale factor that we apply to all font sizes, paddings, etc, and then change it if we determine canvas is too big in first pass? Prob for follow-on work tho; possible no one will run into that issue anyway so not sure we should tackle it until it's a real problem

File: src/cheat_sheet.py (hunk 1 above)
Comment (AndreasArvidsson) [reply src/cheat_sheet.py:50]: I don't know why. Draw just get called multiple times for no reason what I can tell.

**Comment by AndreasArvidsson** [comment]
Yes if the cheat sheet is too large we have to scale every thing down but as you say that might not even be a problem.

**Pull request merged**